2020-pip/RFP.md

```diff
@@ -0,0 +1,278 @@
+
+### Technology
+
+The [pip codebase](https://github.com/pypa/pip) is licensed under the [MIT License](https://github.com/pypa/pip/blob/master/LICENSE.txt). All work submitted or dependencies added must be compliant with this license.
+
+The backend codebase is in Python. The pip documentation is built using reStructuredText and Sphinx.
+
+Potential proposers should be comfortable with Python, and will have support from an additional contractor focused on user interface and user experience design to advise on command line experience improvements.
+
+For Role 1 (Senior developer): Familiarity and expertise with the pip codebase, or with a Python packaging/distribution toolchain codebase, is strongly desired.
+
+For Role 2: Familiarity and expertise with all technologies is not required. Strong Python skills and experience are a must, though. Experience with some automated testing tools (not necessarily the ones we use) are strongly desired.
+
+#### Specific Technologies Used
+
+pip is a Python program that runs on client computers as part of [the Python packaging and distribution toolchain](packaging.python.org/). Understanding of and familiarity with how Python packaging works, how packages are built and installed, and so on, will be helpful.
+
+##### pip codebase
+
+For the best primer, see the [developer documentation](https://pip.pypa.io) for pip.
+
+You can also see the [complete codebase](https://github.com/pypa/pip) on GitHub.
+
+ - [Python](https://www.python.org/) 2.7 and 3.5+ on Linux, Mac OS, and Windows
+ - [pytest](http://pytest.readthedocs.io/)
+ - [tox](https://tox.readthedocs.io/en/latest/)
+ - [Sphinx](http://www.sphinx-doc.org/)
+
+##### Backend and Testing Infrastructure
+
+pip has an extensive continuous integration test suite. This test suite is built using:
+
+- [Python](https://www.python.org/) 2.7 and 3.5+ on Linux, Mac OS, and Windows
+- [tox](https://tox.readthedocs.io/en/latest/)
+- [nox](https://nox.thea.codes/en/stable/)
+- [pre-commit](https://pre-commit.com)
+- [pytest](http://doc.pytest.org/en/latest/)
+
+As well as many other associated Python testing, automation, and code quality tools.
+
+Components of the test suite are executed using multiple services including:
+
+- [Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/index?view=azure-devops)
+- [GitHub Actions](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions)
+- [AppVeyor](https://www.appveyor.com/docs/)
+- [TravisCI](https://docs.travis-ci.com)
+- [Read the Docs](https://docs.readthedocs.io/en/stable/index.html)
+
+### Project Management and Reporting
+
+This project will be led and managed by the Python Software Foundation Director of Infrastructure and an external project manager.
+
+Regular meetings (at most 30 minutes each, 4 times per month) will be held to coordinate efforts among the project managers, backend developers, and UX researcher/designer.
```

Comment:
I guess this "backend" is also extraneous?

```suggestion
Regular meetings (at most 30 minutes each, 4 times per month) will be held to coordinate efforts among the project managers, developers, and UX researcher/designer.
```

Reply:
Yes, thanks, fixed.